src/cedar.js

```diff
@@ -357,6 +358,9 @@ Cedar.prototype.show = function(options){
     if(options.autolabels !== undefined && options.autolabels !== null){
       this.autolabels = options.autolabels;
     }
+    // check if truncate label length has been passed in
+    this.truncateLabelLength = options.truncateLabelLength || 20;
```

Comment:
why default to 20? should just default to `undefined` and if it's falsey, don't do it, right?